localization/english/wc_traits_l_english.yml

```diff
@@ -447,15 +450,15 @@
+ trait_creature_highmountain_tauren_desc:1 "The Highmountain Tauren are an offshoot of tauren descended from Huln Highmountain, who was an ally of the Kaldorei Resistance during the War of the Ancients. During the war, Huln saved Eche'ro, a moose favored by Malorne and was in turn granted a blessing from Cenarius. The blessing granted them the Horns of Eche'ro: moose-like antlers."
  trait_creature_highmountain_tauren_character_desc:0 "This character is a Highmountain Tauren. $trait_creature_highmountain_tauren_desc$"
 
  trait_creature_drogbar:0 "Drogbar"
- trait_creature_drogbar_desc:1 "The Drogbar are a race created by the Titans by modifying the Trogg. While the they were formerly slaves to the black dragon Neltharion, the Drogbar were freed by Huln Highmountain after the War of the Ancients. They are particularly adept in earth magic, and as such they use rocks for most purposes. With their magic rocks are carved to be used for furniture, containers, and even instruments like drums. They are even masters at Jewelcrafting; in particular the chief shaman of a tribe usually wears a neckless of heave gems."
+ trait_creature_drogbar_desc:1 "The drogbar are a race created by the Titans by modifying the trogg. While they were formerly slaves to the black dragon Neltharion, the drogbar were freed by Huln Highmountain after the War of the Ancients. They are particularly adept in earth magic, and as such rocks are used for most purposes. Their magic is practised to carve rocks to be utilized as furniture, containers, and even instruments like drums. They are even masters at jewelcrafting; in particular the chief shaman of a tribe typically wears a neckless of heavy gems."
  trait_creature_drogbar_character_desc:0 "$trait_creature_drogbar_desc$"
 
  trait_creature_trogg:0 "Trogg"
- trait_creature_trogg_desc:1 "The troggs are a race of brutish, cave-dwelling humanoids, originally descendants of the first type of titan-forged manufactured by the Forge of Wills whose designs proved too complex. The result was the stone-skinned brutes known thereafter as "Troggs." Known for their inferior intelligence, they often come into conflict with their distant relatives, the dwarves, as a result."
+ trait_creature_trogg_desc:1 "The troggs are a race of brutish, cave-dwelling humanoids, originally descendants of the first type of titan-forged manufactured by the Forge of Wills whose designs proved too complex. The result was the stone-skinned brutes known thereafter as "Troggs". Known for their inferior intelligence, they often come into conflict with their distant relatives, the dwarves, as a result."
```

Comment:
Why did you change this? In English the standard is that if a sentence ends with a quote the punctuation is inside the quote.

Also you changed other instances of trogg to lower case, but you missed one here.

Reply:
> Why did you change this? In English the standard is that if a sentence ends with a quote the punctuation is inside the quote.

True.

Reply:
I actually wasn't aware of that rule for quotation and punctuation, you're absolutely right there.